src/app/space/create/apps/apps.module.ts

```diff
@@ -33,7 +36,10 @@ const declarations = USE_RUNTIME_CONSOLE ?
 
 const providers = USE_RUNTIME_CONSOLE ?
   [] :
-  [BsDropdownConfig, DeploymentsService];
+  [
+    BsDropdownConfig,
+    { provide: DEPLOYMENTS_SERVICE, useClass: DeploymentsService }
```

Comment:
I'm confused about why you'd need to do this here - we should only have one DeploymentsService in the application.  If there is another one that needs to be removed, please either use a new name or reuse the old one (unless it's in runtime-console).  This is not a pattern we'd like to follow for or providers.

Reply:
Okay, fair enough. There are no naming conflicts, I just considered this to be a code quality thing since we already had the IDeploymentsService extracted out. I'll close this PR as unnecessary.